jujugui/static/gui/src/app/components/header-search/header-search.js

```diff
@@ -47,7 +47,7 @@ YUI.add('header-search', function() {
     _activeForComponent: function() {
       var component = this.props.getAppState(
         'current', 'sectionC', 'component');
-      return !!component && component !== 'profile';
+      return !!component && component !== 'profile' && component !== 'deploy';
```

Comment:
Maybe this should be converted to be 'truthy' as in "we only want to show it for these components...."